src/javascripts/components/mobile-navigation.js

```diff
@@ -25,11 +25,13 @@ MobileNav.prototype.bindUIEvents = function () {
 
       $navToggler.classList.remove(navTogglerActiveClass)
       $navToggler.setAttribute('aria-expanded', 'false')
+      $navToggler.setAttribute('aria-label', 'Show top-level navigation')
```

Comment:
I think that the accessible name of 'menu' alongside the state communication from `aria-expanded` makes this additional label redundant, since it should already say 'menu, expanded' for example.